GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Utils/ServerURLs.cs

```diff
@@ -0,0 +1,7 @@
+public static class ServerURLs{
```

Comment:
"ServerUrls  
{"

Reformat all scripts between merge.

Reply:
Yep, going to look into it for adding the Editor code